cmd/api-server-go/api-server.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/something-to-start-with/api-server-go/internal/handler"
+)
+
+func setupRouter() *gin.Engine {
+	router := gin.Default()
+
+	handler.Routes(router)
+
+	return router
+}
+
+func main() {
+	router := setupRouter()
+	err := router.Run()
+	if err != nil {
+		return
```

Comment:
Why return nothig?

Reply:
main returns nothing
but still I refactored this place so as not to omit the error